dbschema/migrations/versions/e563082c2ef6_.py

```diff
@@ -1,16 +1,16 @@
 """empty message
 
-Revision ID: c90effb6e6d8
+Revision ID: e563082c2ef6
```

Comment:
It seems we are always replacing existing migrations so that only a single migration script exists (from nothing to full schema). This is fine just now since we don't have a production database to upgrade each time we push an update, but in future this will change. It might be a good habit to start generating migrations from the most recent schema version, so we can see if any problems arise

Reply:
Right, I didn't want to have a very long list of migrations while we are developing it, but maybe it's a good idea to start